open-api/rest-catalog-open-api.yaml

```diff
@@ -766,12 +752,20 @@ components:
       properties:
         overrides:
           type: object
+          items:
```

Comment:
Is this valid? I thought it `additionalProperties` was used to set the type of values in a map/object.